pkg/buganalysis/cache.go

```diff
@@ -37,7 +59,10 @@ type bugCache struct {
 	lastUpdateError error
 }
 
-func NewBugCache() BugCache {
+func NewBugCache(skipBugLookup bool) BugCache {
+	if skipBugLookup {
+		return &dummyBugCache{}
+	}
```

Comment:
I think we should let the caller decide which bug cache they need.
so the caller calling
```
if skipBugCache {
  NewDummyBugCache()
} else {
  NewBugCache()
}
```

this allows the actual bug cache implementation `new` function to add options later on for initialization and not have this unrelated argument in there.

Reply:
yeah that was my first version.  i'm fine w/ going back to that.

will update.